apps/site/src/lib/theme/components/accordion/accordion.module.scss

```diff
@@ -0,0 +1,19 @@
+.accordion-trigger {
```

Comment:
Nitpick [repeating]: we generally have uppercase file names (Accordion.module.scss, Button.module.scss) for components to distinguish them from other non-component files.

Reply:
Thanks for catching this! The file name was an error.